hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/compress/TestCodec.java

```diff
@@ -667,7 +673,7 @@ private static void createMapFile(Configuration conf, FileSystem fs, Path path,
     writer.close();
   }
 
-  public static void main(String[] args) throws IOException {
+  /*public static void main(String[] args) throws IOException {
```

Comment:
Leave this uncommented, or potentially remove it?

Reply:
Thanks for the suggestion! I will improve it.